### ISSUE TITLE
Stereochemistry is lost when converting from SMILES to SDF in Python

### ISSUE THREAD

**Issue by skearnes** [rdkit/rdkit#303]
When converting from isomeric SMILES to SDF, stereochemistry information is lost even when using the `includeStereo` flag with `Chem.MolToMolBlock`.

Here's an example:

``` python
from rdkit import Chem

mol = Chem.MolFromSmiles('CC(C)(C)NC[C@@H](C1=CC(=C(C=C1)O)CO)O')
new = Chem.MolFromMolBlock(Chem.MolToMolBlock(mol, includeStereo=True))

print "From SMILES\n-----------"
for atom in mol.GetAtoms():
    print atom.GetChiralTag()

print "\nFrom SDF\n--------"
for atom in new.GetAtoms():
    print atom.GetChiralTag()
```

The output is:

```
From SMILES
-----------
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_TETRAHEDRAL_CW
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED

From SDF
--------
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
CHI_UNSPECIFIED
```

**Comment by coleb** [comment]
I agree MolFromSmiles should not compute 2D coordinates, that is too expensive. 

However we were just bitten by MolToMolBlock not writing out stereo information when reading from a stereo SMILES, even when 'includeStereo=True'. I would propose that when MolToMolBlock's includeStereo flag is 'True',  MolToMolBlock should automatically call Compute2DCoords on the molecule. 

I would even go so far as proposing that 'includeStereo' should default to 'True', and MolToSmiles isomericSmiles should default to 'True' as well. But I could see that as "backwards breaking" and include that as part of those changes.

**Comment by coleb** [comment]
Pinging this again as I just ran into this again this morning. Also spent half a day last week helping a modeler debug a stereochemistry issue that came down to RDKit not saving stereo chemistry by default when converting from SMILES to SDF. 

I think it comes under the "we should be saving stereo by default" everywhere strategy that I think we should switch over to.

**Comment by greglandrum** [comment]
@coleb : thanks for ping this again
I think it makes sense to make the suggested changes as part of the "make the default arguments more correct" effort that I want to do for the next release. 
I'll start putting together a branch for that.
